priv/templates/boot.eex

```diff
@@ -272,6 +272,9 @@ fi
 # By default it's RELEASE_ROOT_DIR
 if [ -z "$RELEASE_CONFIG_DIR" ]; then
     RELEASE_CONFIG_DIR="$RELEASE_ROOT_DIR"
+    CUSTOM_CONFIG_DIR=""
+else
+    CUSTOM_CONFIG_DIR="true"
```

Comment:
It's to track whether a custom config dir is in use (i.e. `RELEASE_CONFIG_DIR` was set before the script got to this point). Will change to `HAS_CUSTOM_CONFIG_DIR`.